rivendale/ethsigns
# feat (zodiac-sign): verify sign mined the sign

**What does this PR do?**

- create user account
- list user signs
- verify sign ownership

**Description of Task to be completed?**

- create user account
- list user signs
- verify sign ownership

**How should this be manually tested?**

- Clone this repo using the command `git clone https://github.com/rivendale/ethsigns.git`
- Create a local copy of this branch by running `git fetch ft-verify-sign-ownership-41`
- Checkout to the branch `git checkout ft-verify-sign-ownership-41`
- Start the application with the command: `flask run`
- Navigate to the documentation `{base-url}/api/v1/`

**Any background context you want to provide?**

N/A

**What are the relevant GitHub issue**

resolves #41 

**Screenshots (if appropriate)**

N/A

**Questions:**

N/A
